warble/functions.cc

```diff
@@ -156,19 +164,40 @@ bool Profile(Database* db, Any req, Any* rep) {
       db->get("_followers_" + username);
   if (followers) {
     for (std::string f : *followers) {
-      std::string* s = reply.add_followers();
+      std::string *s = reply.add_followers();
       *s = f;
     }
   }
   std::optional<std::vector<std::string>> following =
       db->get("_following_" + username);
   if (following) {
     for (std::string f : *following) {
-      std::string* s = reply.add_following();
+      std::string *s = reply.add_following();
       *s = f;
     }
   }
   rep->PackFrom(reply);
   LOG(INFO) << "Got profile for user " << username;
   return true;
+}
+
+bool Stream(Database *db, Any req, Any *rep) {
+  StreamRequest request;
+  StreamReply reply;
+  req.UnpackTo(&request);
+  std::string hashtag = request.hashtag();
+  std::optional<std::vector<std::string>> exists =
+      db->get("_hashtag_" + hashtag);
+  if (exists) {
+    for (auto s : (*exists)) {
+      warble::Warble *w = reply.add_warbles();
```

Comment:
ideally would not send the warble if it has already been streamed to a particular client. may be difficult to implement as is, but would be easier if this were called only when new warbles are streamed (see earlier comment about polling)